.github/workflows/publish.yml

```diff
@@ -14,6 +13,7 @@ jobs:
     uses: ./.github/workflows/test.yml
   publish:
     needs: test
+    if: ${{ github.ref_name == 'master' }}
```

Comment:
I don't think this is necessary. In fact, we _could_ decide to maintain multiple branches, for example if we went 3.0 on `master` but still wanted to allow bugfixes to be released to 2.X.